starters/solidjs-tailwind/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "description": "SolidJS and TailwindCSS",
+  "author": "Maarten, Chinedu, Daina, Mattai, Mark, Victor, Dane, Jan, Jerry",
```

Comment:
Hi @hdJerry, seeing the word "author" is singular and package.json offering another field called "contributors" would it make sense to switch to that one instead? I believe it takes an array so we can create an entry for everyone.

```json
{
   "contributors": [
     "Jerry Hogan <jerry.hogan@thisdot.co>",
     "Chinedu Okpala <chinedu.okpala@thisdot.co>",
     "Mattia \"Mattai\" Magi <mattia.magi@thisdot.co>,
     "..."
   ]
}
```

Reply:
Okay, let me try this. Thanks